src/core_atmosphere/physics/mpas_atmphys_vars.F

```diff
@@ -893,6 +925,12 @@ module mpas_atmphys_vars
     frc_urb_p,        &!urban fraction                                                                         [-]
     ust_urb_p          !urban u* in similarity theory                                                        [m/s]
 
+!=================================================================================================================
+!.. variables and arrays related to the Noahmp land-surface parameterization:
+!=================================================================================================================
+
+ type(NoahmpIO_type):: mpas_noahmp
+
 !-- arrays needed to run RUC LSM
```

Comment:
may want to addd the same style, i.e.:
!===========================================
before and after this line